.cloud-repo-tools.json

```diff
@@ -9,5 +9,21 @@
   "requiresKeyFile": true,
   "requiresProjectId": true,
   "client_reference_url": "https://cloud.google.com/nodejs/docs/reference/text-to-speech/latest/",
-  "release_quality": "alpha"
+  "release_quality": "alpha",
```

Comment:
1. Upgrade repo-tools to 2.2.6
2. Delete the following fields from this file: `_product`, `id`, `name`, `short_name`, `docs_url`, `description`, `api_id`.
3. Add the following to this file: `"product": "tts"`
4. Re-run `npm run generate-scaffolding`

Reply:
Done